content/errors/codes.textile

```diff
@@ -0,0 +1,717 @@
+---
+title: Error codes
+meta_description: "Understand Ably error codes and their causes, to resolve them efficiently."
+languages:
+  - javascript
+---
+
+Use the error codes provided, which include descriptions, possible causes, and solutions, to troubleshoot issues effectively.
+
+h2(#500). 500: Internal error
```

Comment:
Do we really have 500 error code, I was under impression that all AblyError codes are {HttpStatusCode}XX